Assets/MixedRealityToolkit/Utilities/Scripts/Editor/EnforceEditorSettings.cs

```diff
@@ -52,6 +53,22 @@ static EnforceEditorSettings()
                 }
             }
 
+            if (PlayerSettings.scriptingRuntimeVersion != ScriptingRuntimeVersion.Latest)
+            {
+                if (EditorUtility.DisplayDialog(
+                        "Change the Scripting Runtime Version to 4.x?",
+                        "The Mixed Reality Toolkit would like to change the Scripting Runtime Version to use .NET 4.x.\n\n" +
+                        "In order for the change to take place the Editor must be restarted.\n\n" +
+                        "WARNING: If you do not make this change, then it's likely you're build will not complete successfully.\n\n" +
```

Comment:
you're -> your.

Reply:
What makes builds "likely" vs "will" fail? I don't want developers to think of this as the kid who cried wolf.

Reply:
Thanks! I'll be sure to clean up the language a bit.